README.md

```diff
@@ -4,27 +4,121 @@ Translate is a library for machine translation written in PyTorch. It provides t
 
 ## Requirements and Installation
 
-Translate requires
+### Translate requires:
 * A Linux operating system with a CUDA compatible card
 * C++ compiler supporting ECMAScript syntax for `<regex>`, such as GCC 4.9 and above
 * A [CUDA installation](https://docs.nvidia.com/cuda/). We recommend CUDA 8 or CUDA 9
 
-To install Translate from source:
-- Install Anaconda or Miniconda3 from https://conda.io/miniconda.html:
-  ```
+### To install Translate from source:
+These instructions were mainly tested on CentOS 7.4.1708 with a Tesla M40 card
+and a CUDA 8 installation. We highly encourage you to [report an issue](https://github.com/pytorch/translate/issues)
+if you are unable to install this project for your specific configuration.
+
+- If you don't already have an existing [Anaconda](https://www.anaconda.com/download/)
+environment with Python 3.6, you can install one via [Miniconda3](https://conda.io/miniconda.html):
+  ```bash
   wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
   chmod +x miniconda.sh
   ./miniconda.sh -b -p ~/miniconda
   rm miniconda.sh
   . ~/miniconda/bin/activate
   ```
-- Please refer to the `install.sh` script. In many cases you can simply run `bash install.sh`. We have tested this script on CentOS 7.4.1708 with a Tesla M40 card and a CUDA 8 installation. We encourage you to report an [issue](https://github.com/pytorch/translate/issues) if you are unable to install this project for your specific configuration.
 
-Alternatively, you can launch an AWS instance using the `pytorch_translate_initial_release` image (AMI ID: ami-04ff53cdd573658dc). Once you have ssh'ed to the AWS instance, the example commands below should work after running `cd translate`.
+- Clone the Translate repo:
+  ```bash
+  git clone --recursive https://github.com/pytorch/translate.git
```

Comment:
We don't have submodules AFAIK

Reply:
I was just thinking it's safer to have that option even if it's not needed (in case we go back to them again) - and it's less cognitive load to not have to remember which of pytorch/translate, pytorch/pytorch, or onnx we need to use --recursive with when git cloning.

Reply:
Users are copying the instructions so I don't see the cognitive load point. I wouldn't put an option that is useless.

Reply:
Removing